aslam_cv/aslam_cameras/include/aslam/cameras/implementation/PinholeProjection.hpp

```diff
@@ -777,9 +778,14 @@ bool PinholeProjection<DISTORTION_T>::initializeIntrinsics(const std::vector<Gri
     }
   }
 
-  //get the median of the guesses
-  if(f_guesses.empty())
-    return false;
+  // Get the median of the guesses if available.
+  if(f_guesses.empty()) {
+   float input_guess;
+   std::cout << "Initialization of focal length failed. Provide manual initialization: ";
+   std::cin >> input_guess;
+   std::cout << "Initialize focal length to " << input_guess << "\n";
+   f_guesses.push_back(static_cast<float>(input_guess));
```

Comment:
`emplace_back`?

Reply:
Trying to stay conistent with https://github.com/ethz-asl/kalibr/blob/a5186202b7bf1db7d9e0828ef24dff2bba43340d/aslam_cv/aslam_cameras/include/aslam/cameras/implementation/PinholeProjection.hpp#L776